<launch>
  <param name="robot_description" command="cat $(find smart_description)/urdf/smart.urdf.xacro" />

  <rosparam file="$(find smart_hardware_interface)/config/hardware.yaml" command="load"/>
  <rosparam file="$(find smart_hardware_interface)/config/controllers.yaml" command="load"/>
  <!-- rosparam file="$(find smart_hardware_interface)/config/joint_limits.yaml" command="load"/ -->

  <node name="smart_hardware_interface" pkg="smart_hardware_interface" type="smart_hardware_interface_node" output="screen"/>

  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" args="
                /controller/state
                /controller/base
                " />


</launch>
